AnkiDroid/src/main/java/com/ichi2/anki/jsaddons/NpmUtils.kt

```diff
@@ -83,10 +83,7 @@ object NpmUtils {
             val arr = name.split('/')
             val scope = arr[0].removePrefix("@")
             val isValidScopeName = validateName(scope)
-
-            if (!isValidScopeName) {
-                return isValidScopeName
-            }
+            if (!isValidScopeName) return isValidScopeName
```

Comment:
The proposed fix didn't fix the warning

```suggestion
            if (!isValidScopeName) return false
```

Reply:
Done .